google/cloud/forseti/common/util/email/base_email_connector.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base email connector to select connector"""
+
+import abc
+import base64
+import os
+import jinja2
+from sendgrid.helpers import mail
```

Comment:
Add a blank line here.  This is a 3rd party library, so that it needs to be grouped separately from the built-in libraries.

Reply:
Done